ev3dev2/motor.py

```diff
@@ -2182,6 +2189,138 @@ def turn_left(self, speed, degrees, brake=True, block=True):
         """
         self._turn(speed, abs(degrees) * -1, brake, block)
 
+    def odometry_coordinates_log(self):
+        log.info("%s: odometry angle %s at (%d, %d)" %
+            (self, math.degrees(self.theta), self.x_pos_mm, self.y_pos_mm))
+
+    def odometry_start(self, theta_degrees_start=0.0, x_pos_start=0.0, y_pos_start=0.0):
```

Comment:
What do you think about breaking this out into a separate `WheelOdometry` class which can also be used independently? As in, you could instantiate a class instance and call `.start()` and it would track position for you? This class would then store an instance of that odom class when needed.

Reply:
The reason being I don't think we have any other classes where we could do odometry.

Reply:
If you'd prefer it within this class, that's fine with me :+1: